modules/comment-likes.php

```diff
@@ -0,0 +1,185 @@
+		if ( 'comment_likes' !== $column_name ) {
+			return;
+		}
+
+		$blog_id = Jetpack_Options::get_option( 'id' );
+
+		$permalink = get_permalink( get_the_ID() );
+		?>
+		<a title=""
+		   data-comment-id="<?php echo (int) $comment_id; ?>"
+		   data-blog-id="<?php echo (int) $blog_id; ?>"
+		   class="comment-like-count"
+		   id="comment-like-count-<?php echo (int) $comment_id; ?>"
+		   href="<?php echo esc_url( $permalink ); ?>#comment-<?php echo (int) $comment_id; ?>"
+		>
+			<span class="like-count">0</span>
+		</a>
+		<?php
+	}
+
+	function enqueue_admin_styles_scripts() {
+		wp_enqueue_style( 'comment-like-count', plugins_url( 'comment-likes/admin-style.css', __FILE__ ), array(), JETPACK__VERSION );
+		wp_enqueue_script( 'comment-like-count', plugins_url( 'comment-likes/comment-like-count.js', __FILE__ ), array( 'jquery' ), JETPACK__VERSION );
+	}
+
+	public function add_like_count_column( $columns ) {
+		$columns['comment_likes'] = '<span class="vers"><img title="' . esc_attr__( 'Comment likes', 'jetpack' ) . '" alt="' . esc_attr__( 'Comment likes', 'jetpack' ) . '" src="//s0.wordpress.com/i/like-grey-icon.png" /></span>';
+
+		return $columns;
+	}
+
+	public function frontend_init() {
+		if ( is_admin() ) {
+			return;
+		}
+
+		add_action( 'wp_enqueue_scripts', array( $this, 'load_styles_register_scripts' ) );
+		add_filter( 'comment_text', array( $this, 'comment_likes' ), 10, 2 );
+	}
+
+	public function load_styles_register_scripts() {
+		wp_enqueue_style( 'jetpack_likes', plugins_url( 'likes/style.css', __FILE__ ), array(), JETPACK__VERSION );
+		wp_enqueue_script( 'postmessage', plugins_url( '_inc/postmessage.js', dirname(__FILE__) ), array( 'jquery' ), JETPACK__VERSION, false );
+		wp_enqueue_script( 'jetpack_resize', plugins_url( '_inc/jquery.jetpack-resize.js' , dirname(__FILE__) ), array( 'jquery' ), JETPACK__VERSION, false );
+		wp_enqueue_script( 'jetpack_likes_queuehandler', plugins_url( 'likes/queuehandler.js' , __FILE__ ), array( 'jquery', 'postmessage', 'jetpack_resize' ), JETPACK__VERSION, true );
+	}
+
+	public function comment_likes( $content, $comment = null ) {
+		if ( empty( $comment ) ) {
+			return $content;
+		}
+
+		if ( ! $this->settings->is_likes_visible() ) {
+			return $content;
+		}
+
+		$blog_id   = Jetpack_Options::get_option( 'id' );
+		$url       = home_url();
+		$url_parts = parse_url( $url );
+		$domain    = $url_parts['host'];
+
+		$comment_id = get_comment_ID();
+		if ( empty( $comment_id ) && ! empty( $comment->comment_ID ) ) {
+			$comment_id = $comment->comment_ID;
+		}
+
+		if ( empty( $content ) || empty( $comment_id ) ) {
+			return $content;
+		}
+
+		// In case master iframe hasn't been loaded. This could be the case when Post Likes module is disabled,
+		// or on pages on which we have comments but post likes are disabled.
+		if ( ! has_action( 'wp_footer', 'jetpack_likes_master_iframe' ) ) {
+			add_action( 'wp_footer', 'jetpack_likes_master_iframe', 21 );
+		}
+
+		$uniqid = uniqid();
```

Comment:
Might this better be done as `wp_generate_uuid4()` ?

https://github.com/WordPress/WordPress/blob/11bae4ded306934df183781bbdd7a14c70e6245c/wp-includes/functions.php#L5613-L5628